apps/publish/published_item.py

```diff
@@ -170,8 +172,11 @@ def on_update(self, updates, original):
 
     def on_updated(self, updates, original):
         if 'marked_for_user' in updates:
+            updated = copy(original)
```

Comment:
that's like `updated = original.copy()` - or did you mean to use `copy.deepcopy()` ?

Reply:
Yup, it's a shallow copy.